setup.py

```diff
@@ -17,7 +17,7 @@
 
 DISTNAME = 'celer'
 DESCRIPTION = descr
-with open('README.md', 'r') as f:
+with open('README.md', 'r', encoding='utf-8') as f:
```

Comment:
@mathurinm,
didn't check that yet, but I trust this will solve the problem of uploading non-ASCII characters
using windows VM